core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,16 +18,40 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
```

Comment:
I think these clasess (top-navigation-bar-directive and oppia-logo) comes from a directive. So maybe, you can add you CSS.